order-status/README.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+This document explains the structure of a STAT **Order Status** request. Operation returns a list/array of statuses that have affected an order.  The most recent status code is the current status of the order.
+
+
+Endpoint: GET /order/{id}/status
```

Comment:
Do we need to support pagination / limiting / sorting / filtering?

Reply:
suggestion that this is an optional extension, as filtering / paging mechanisms could differ by provider